Expensify/App
# Code Block Formatting Bug

### Details
I have fixed the bug where text in code blocks breaks words and symbols. It will now no longer do so. 

### Fixed Issues
[https://github.com/Expensify/Expensify.cash/issues/993](Issue 993)

### Tests
For example:
1. Sign into the Expensify.cash app
2. Start a chat with some other user
3. Use three backticks (```) follows by a newline (Shift+Enter) to add a block of long code, ending if with another newline (Shift+Enter) and three more backticks (```).
4. See the output of the message is formatted correctly

### Tested On


### Screenshots

#### Web



#### Mobile Web


#### Desktop
<img width="1312" alt="Screen Shot 2020-12-30 at 12 18 45 PM" src="https://user-images.githubusercontent.com/15069998/103399112-86040300-4b0d-11eb-9408-53e98e4ea1ef.png">


#### iOS
<img width="562" alt="Screen Shot 2020-12-30 at 12 16 22 PM" src="https://user-images.githubusercontent.com/15069998/103399144-961be280-4b0d-11eb-8178-127d685e60b6.png">


#### Android
